index.bs

```diff
@@ -3458,18 +3457,18 @@ The following abstract operations support the implementation of the
      queue entry/byte length=]).
   1. Let |destStart| be |pullIntoDescriptor|'s [=pull-into descriptor/byte offset=] +
      |pullIntoDescriptor|'s [=pull-into descriptor/bytes filled=].
-  1. Assert: ! [$CanCopyDataBlockBytes$](|pullIntoDescriptor|'s [=pull-into descriptor/buffer=],
-     |destStart|, |headOfQueue|'s [=readable byte stream queue entry/buffer=],
+  1. Let |descriptorBuffer| be |pullIntoDescriptor|'s [=pull-into descriptor/buffer=].
+  1. Let |queueBuffer| be |headOfQueue|'s [=readable byte stream queue entry/buffer=].
+  1. Assert: ! [$CanCopyDataBlockBytes$](|descriptorBuffer|, |destStart|, |queueBuffer|,
      |headOfQueue|'s [=readable byte stream queue entry/byte offset=], |bytesToCopy|) is true.
```

Comment:
"headOfQueue's byte offset" appears both in the `CanCopyDataBlockBytes` assertion and in the `CopyDataBlockBytes` call below. Should we also introduce a variable for it?